fdb-record-layer-lucene/src/main/java/org/apache/lucene/codecs/lucene84/LuceneOptimizedPostingsReader.java

```diff
@@ -0,0 +1,2006 @@
+/*
+ * LuceneOptimizedPostingsReader.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2022 Apple Inc. and the FoundationDB project authors
```

Comment:
If this is taken from code that was originally in Lucene and then modified, I think we need a slightly different attribution header here. I thought there was an example somewhere of a place we've done this, but I'm not actually sure that there is

Reply:
Could not find it..